exosphere-shared/src/compile-service-routes.ls

```diff
@@ -8,12 +8,13 @@ module.exports = (app-config, base-path) ->
   service-routes = []
   for protection-level of app-config.services
     for role, service-data of app-config.services["#{protection-level}"]
-      service-config = yaml.safe-load fs.read-file-sync(path.join(base-path ? process.cwd!, service-data.location, 'service.yml'), 'utf8')
-      service-routes.push do
-        {
-          role: role
-          receives: service-config.messages.receives
-          sends: service-config.messages.sends
-          namespace: service-data.namespace
-        }
+      if service-data.location
```

Comment:
We run `docker run --rm=true #{image} cat service.yml`. `compile-service-routes.ls` was being used on local services only anyways, this just centralizes the `if service-data.location` check and makes this function more robust.